src/platform/CMakeLists.txt

```diff
@@ -18,8 +18,8 @@ project(osconfig-platform)
 set(osconfig_platform_files
     ./Log.c
     ./Main.c
-    ./ManagementModule.cpp
-    ./ModulesManager.cpp
+    ./MmiClient.c
+    ./ModuleManager.c

```

Comment:
>/ModuleManager.c [](http://example.com/codeflow?start=5&length=16)

It's "ModulesManager", not "ModuleManager"

[https://github.com/Azure/azure-osconfig/blob/main/docs/architecture.md#44-modules-manager](https://github.com/Azure/azure-osconfig/blob/main/docs/architecture.md#44-modules-manager)